pkg/apis/servicecatalog/testapi/testapi.go

```diff
@@ -30,62 +30,19 @@ import (
 	"reflect"
 	"strings"
 
+	"github.com/kubernetes-incubator/service-catalog/pkg/api"
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
 	"k8s.io/apimachinery/pkg/api/meta"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/runtime/serializer/recognizer"
-	"k8s.io/client-go/pkg/api"
```

Comment:
not sure what this file is, but cool to get rid of code.

Reply:
@MHBauer we are mixing versioned and unversioned api packages, but in 1.8 both will be completely removed and only the versioned one is present in https://github.com/kubernetes/api/tree/master/core/v1
Hence the preparatory cleanup

Reply:
re: getting rid of code
This file is used for serialization, fuzzer and rest client tests IIRC.
The code seem to be copy-pasted from main kube repo without thinking much - 90% of code was not used by our tests and was referencing core kube resources we don't need. explicitly left only our Service Catalog resources.